### PR TITLE
Open Mobile Map Package Sample

### PR DESCRIPTION
**Pull request by tschie**
Demonstrates opening a mobile map package.

### CODE REVIEW

**Comment by tschie** [comment]
@john0005 Can you please review?

**Comment by john0005** [comment]
Looks like Yellowstone needs to be added to the samples data. I refreshed it and didn't see it in there.

File: src/main/java/com/esri/samples/map/open_mobile_map_package/OpenMobileMapPackageSample.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          //add the map from the mobile map package to the map view
+          mapView.setMap(mobileMapPackage.getMaps().get(0));
+        } else {
+          Alert alert = new Alert(Alert.AlertType.ERROR, "Failed to load the mobile map package");
+          alert.show();
+        }
+      });
+
+      // add the map view to stack pane
+      stackPane.getChildren().addAll(mapView);
```
Comment (john0005) [inline src/main/java/com/esri/samples/map/open_mobile_map_package/OpenMobileMapPackageSample.java]: Just an `add()` is needed, but that being picky. :)

**Review by john0005 — CHANGES_REQUESTED** [review]
Once Yellowstone is in samples data I will retry and it is good to go.

**Comment by tschie** [comment]
@john0005 my bad, the mmpk should be there now. Please merge if it looks good after that.

File: src/main/java/com/esri/samples/map/open_mobile_map_package/OpenMobileMapPackageSample.java (hunk 1 above)
Comment (tschie) [reply src/main/java/com/esri/samples/map/open_mobile_map_package/OpenMobileMapPackageSample.java]: picky is good

**Comment by john0005** [comment]
Looks good!

**Pull request merged**